src/booleans.js

```diff
@@ -1,62 +1,109 @@
 function negate(a) {
-  // your code here
-};
+  if (a === false) {
+    return true;
+  }
+  return false;
+}
 
 function both(a, b) {
-  // your code here
-};
+  if (a === true && b === true) {
+    return true;
+  }
+  return false;
+}
 
 function either(a, b) {
-  // your code here
-};
+  if (a === true || b === true) {
+    return true;
+  }
+  return false;
+}
 
 function none(a, b) {
-  // your code here
-};
+  if (a === false && b === false) {
+    return true;
+  }
+  return false;
+}
 
 function one(a, b) {
-  // your code here
-};
+  if ((a === true && b === false) || (a === false && b === true)) {
+    return true;
+  }
+  return false;
+}
 
 function truthiness(a) {
-  // your code here
-};
+  if (a) {
+    return true;
+  }
+  return false;
```

Comment:
There is a shorter way to do this, see the containsVowels function for a detailed explanation on what you could do here